.github/workflows/prepare-release.yml

```diff
@@ -93,178 +93,202 @@ jobs:
 
     if: |
-      startsWith(github.event.issue.title, '[release request] ')
-      && github.event.issue.pull_request == null
-      && needs.automation.outputs.enabled
-      && (
-          (github.event_name == 'issues')
+      startsWith(github.event.issue.title, '[release request] ') &&
+      github.event.issue.pull_request == null &&
+      needs.automation.outputs.enabled &&
+      (
+        (github.event_name == 'issues')
         ||
-          (github.event_name == 'issue_comment'
-          && github.event.issue.state == 'open'
-          && contains(github.event.comment.body, '/PrepareRelease')
-          && github.event.comment.user.login != needs.automation.outputs.username)
+        (github.event_name == 'issue_comment' &&
+         github.event.issue.state == 'open' &&
+         contains(github.event.comment.body, '/PrepareRelease') &&
+         github.event.comment.user.login != needs.automation.outputs.username)
       )
 
     env:
       GH_TOKEN: ${{ secrets[needs.automation.outputs.token-secret-name] }}
 
     steps:
-    - name: check out code
+    - name: Check out code
       uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2
       with:
-        token: ${{ secrets[needs.automation.outputs.token-secret-name] }}
+        token: ${{ env.GH_TOKEN }}
 
     - name: Process release request issue being opened or commented
       shell: pwsh
+      env:
+        BOT_USER_EMAIL: ${{ needs.automation.outputs.email }}
+        BOT_USER_NAME: ${{ needs.automation.outputs.username }}
+        ISSUE_BODY: ${{ github.event.issue.body }}
+        ISSUE_NUMBER: ${{ github.event.issue.number }}
+        REQUESTOR_USER_NAME: ${{ github.event.comment.user.login || github.event.sender.login }}
+        TARGET_BRANCH: ${{ github.event.repository.default_branch }}
       run: |
         Import-Module .\build\scripts\prepare-release.psm1
 
         TagCodeOwnersOnOrRunWorkflowForRequestReleaseIssue `
-          -gitRepository '${{ github.repository }}' `
-          -targetBranch '${{ github.event.repository.default_branch }}' `
-          -triggeringEventName '${{ github.event_name }}' `
-          -approvingGroups '@${{ github.repository_owner }}/dotnet-approvers @${{ github.repository_owner }}/dotnet-maintainers' `
-          -requestedByUserName '${{ github.event.comment.user.login || github.event.sender.login }}' `
-          -issueNumber '${{ github.event.issue.number }}' `
-          -issueBody $env:ISSUE_BODY `
-          -gitUserName '${{ needs.automation.outputs.username }}' `
-          -gitUserEmail '${{ needs.automation.outputs.email }}'
-      env:
-        ISSUE_BODY: ${{ github.event.issue.body }}
+          -gitRepository ${env:GITHUB_REPOSITORY} `
+          -targetBranch ${env:TARGET_BRANCH} `
+          -triggeringEventName ${env:GITHUB_EVENT_NAME} `
+          -approvingGroups "@${env:GITHUB_REPOSITORY_OWNER}/dotnet-approvers @${env:GITHUB_REPOSITORY_OWNER}/dotnet-maintainers" `
```

Comment:
It is still file string concatenation. It will be not safer, I understand, that the change is for the consistency :)